src/ec/packages/ec-component-media-container/media-container.test.js

```diff
@@ -0,0 +1,62 @@
+import path from 'path';
+import { renderTwigFile } from '@ecl-twig/test-utils';
+
+describe('EC - Media container image', () => {
+  const template = path.resolve(__dirname, './media-container.html.twig');
+  test(`- Media container image renders correctly`, done => {
+    expect.assertions(1);
+
+    const defaultDataStructure = {
```

Comment:
Please use adapter spec data. If spec data changes, tests will miss it otherwise.